hack/generate_client.sh

```diff
@@ -20,11 +20,7 @@ rm -rf ./pkg/client/{clientset,informers,listers}
 mv "${TMP_DIR}"/src/github.com/koordinator-sh/koordinator/pkg/client/* ./pkg/client
 
 function custom_sed(){
-    if [[ "$OSTYPE" == "darwin"* ]]; then
-      sed -i '' $@
-    else
-      sed -i $@
-    fi
+  sed -i -e $@
```

Comment:
it's not working :(
![image](https://user-images.githubusercontent.com/24452340/169497045-cb4da1c0-fdd0-4f2a-b9c7-817b16e5f40c.png)

Reply:
awkward